### PR TITLE
Develop into Master

### PR DESCRIPTION
**Pull request by stevieing**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: app/models/label_fields.rb [hunk 1]
```diff
@@ -63,17 +63,17 @@ def initialize(labels)
     ##
     # find value for key
     def find(key)
-      values[key.to_sym]
+      values[key]
     end
 
     ##
-    # Labels is of type OpenStruct
+    # Labels is a hash
```
Comment (harrietc52) [inline app/models/label_fields.rb:70]: Not sure the relevance of this comment for the method?

File: config/environments/development.rb [hunk 2]
```diff
@@ -58,8 +58,6 @@
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
```
Comment (harrietc52) [inline config/environments/development.rb:60]: Might this need to be removed from the deployment project too

**Review by harrietc52 — APPROVED** [review]

File: config/environments/development.rb (hunk 2 above)
Comment (stevieing) [reply config/environments/development.rb:60]: I have just checked and thankfully this does not exist in the environment in the deployment project.

File: app/models/label_fields.rb (hunk 1 above)
Comment (stevieing) [reply app/models/label_fields.rb:70]: I have absolutely no idea.

**Pull request merged**